sdk/communication/azure-communication-common/pom.xml

```diff
@@ -54,6 +54,11 @@
       <version>1.15.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-json</artifactId>
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-json;dependency} -->
+    </dependency>
```

Comment:
Do we need to add this dependency explicitly? Can we use transitive dependency from azure-core?